src/bootstrap/lib.rs

```diff
@@ -541,6 +541,17 @@ impl Build {
         if self.config.llvm_enabled() {
             features.push_str(" llvm");
         }
+
+        // If debug logging is on, then we want the default for tracing:
+        // https://github.com/tokio-rs/tracing/blob/3dd5c03d907afdf2c39444a29931833335171554/tracing/src/level_filters.rs#L26
+        // which is everything (including debug/trace/etc.)
+        // if its off, th
```

Comment:
Looks like this got a bit mangled in the comment.

Reply:
oops, forgot to git add my changes lol